tools/cli/commands.js

```diff
@@ -1579,8 +1579,31 @@ function doTestCommand(options) {
   // cleaned up on process exit. Using a temporary app dir means that we can
   // run multiple "test-packages" commands in parallel without them stomping
   // on each other.
-  var testRunnerAppDir =
-        options['test-app-path'] || files.mkdtemp('meteor-test-run');
+  let testRunnerAppDir;
+  const testAppPath = options['test-app-path'];
+  if (testAppPath) {
+    try {
+      if (files.mkdir_p(testAppPath, 0o700)) {
```

Comment:
Thanks for reviewing @mitar - I did this as `files.mkdtemp` (which we're essentially replacing when a `--test-app-path` value is passed in) [is doing this](https://github.com/meteor/meteor/blob/44796a183111ed18fb27518780bf98267d04e7d5/tools/fs/files.js#L666).